dlgr/griduniverse/experiment.py

```diff
@@ -670,23 +679,23 @@ def instructions(self):
             text += """ Movement commands will be ignored almost all of the time,
```

Comment:
This is eventually going to have to be re-done. I suspect we'll just put game specific instructions in the YAML config instead of trying to automate the documentation generation. Once e.g. `object_count` an `reward` (which should become `calories`) are object/item properties this won't really work.